misc/python/materialize/zippy/source_actions.py

```diff
@@ -44,13 +44,35 @@ def new(self, capabilities: Capabilities) -> list[Action]:
                         cluster_name=random.choice(
                             source_capable_clusters(capabilities)
                         ),
+                        uses_ssh_tunnel=random.choice([True, False]),
                     ),
                 )
             ]
         else:
             return []
 
 
+class AlterSourceParameterized(ActionFactory):
+    """Alters a source in Materialized."""
+
+    @classmethod
+    def requires(cls) -> set[type[Capability]]:
+        return {MzIsRunning, StoragedRunning, KafkaRunning, TopicExists}
```

Comment:
You need `SourceExists` as a requirement here as well.

Reply:
✅